decidim-dev/lib/decidim/dev/test/rspec_support/capybara.rb

```diff
@@ -23,7 +23,7 @@ def switch_to_default_host
   options = ::Selenium::WebDriver::Chrome::Options.new
   options.args << "--headless"
   options.args << "--no-sandbox"
-  options.args << "--window-size=1024,768"
+  options.args << "--window-size=1920,3000"
```

Comment:
why this change?
I would like to stay with 1024 so that we are sure that everything renders ok in mobile devices

Reply:
I'll revert it, I changed it because when a test fails and you get a screen capture sometimes there's not enough information there